package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pool-request",
-  "version": "0.0.0",
+  "version": "0.2.1",
```

Comment:
Definitely not a major version update. But I would have assumed you would have gone off the pull requests